package.json

```diff
@@ -105,5 +106,8 @@
       "import/no-anonymous-default-export": "off"
     }
   },
-  "proxy": "http://localhost:4242"
+  "proxy": "http://localhost:4242",
+  "dependencies": {
+    "swr": "^0.5.5"
```

Comment:
you probably want this as a devDependency.

Reply:
Yes, thanks!